python/ray/_private/runtime_env/context.py

```diff
@@ -49,6 +49,9 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
             executable = f'"{self.py_executable}"'  # Path may contain spaces
         elif language == Language.PYTHON:
             executable = f"exec {self.py_executable}"
+            for i, arg in enumerate(passthrough_args):
```

Comment:
It looks like under Windows, the above branch will be taken instead of this branch. Does this change fix your issue?

Reply:
Ah, I blundered. Must have been too tired when I patched this. I'll fix it in a bit.